### ISSUE TITLE
Test plan for browser extension

### ISSUE THREAD

**Issue by nicksnyder** [sourcegraph/sourcegraph#1703]
Browser Extension Test Plan

- Install the browser extension by following https://docs.sourcegraph.com/integration/browser_extension
Steps are different for Firefox and Chrome, you should check both.
- Navigate to any github.com page and verify the command palette is present and can be used to run extension-provided commands
- Navigate to a code view on github, eg. https://github.com/sourcegraph/sourcegraph/blob/master/shared/src/api/extension/types/selection.ts.
    - Verify “View file” button is present and working
    - Make sure hovers/defs/refs work
- Navigate to a repository view on github, eg. https://github.com/sourcegraph/sourcegraph
    - Verify “View repository” button is present and working
- Navigate to a diff view on github, eg. https://github.com/sourcegraph/sourcegraph/commit/fbb3193c777becdf420be9a94e6cd81d9228828e#diff-c92019313327bf9f16ce5d3780de537d 
    - Make sure hover/defs/refs work
    - Verify “View file (base)” and “View file (head)” buttons are present and working
- Navigate to any gitlab.com page and verify the command palette is present and can be used to run extension-provided commands
- Navigate to a code view on gitlab, eg. https://gitlab.com/gitlab-org/gitlab-runner/blob/master/log/configuration.go.
    - Make sure hovers/defs/refs work.
    - Verify “View file” button is present and working
- Navigate to a diff view on gitlab, eg. https://gitlab.com/gitlab-org/gitlab-runner/commit/1a730d8f4d33607b610868f003fda5ba22e63473.
    - Make sure hovers/defs/refs work.
    - Verify “View file (base)” and “View file (head)” buttons are present and working
- Make a query using the search shortcut by following https://docs.sourcegraph.com/integration/browser_extension#search-shortcut-in-location-bar
- Change the Sourcegraph instance to use by following https://docs.sourcegraph.com/integration/browser_extension#configuring-the-sourcegraph-instance-to-use
    - Verify extension connects to the new instance & still works as expected

**Comment by nicksnyder** [comment]
@vanesa @attfarhan can you please test this and comment when you are done

**Comment by attfarhan** [comment]
Filed https://github.com/sourcegraph/sourcegraph/issues/1741, which I don't think is a release blocker. LGTM otherwise.

**Comment by lguychard** [comment]
@vanesa when do you think you will get a chance to test this?

**Comment by nicksnyder** [comment]
I tested this and filed a couple issues (neither of which are in 3.0-beta milestone).
https://github.com/sourcegraph/sourcegraph/issues/1842
https://github.com/sourcegraph/sourcegraph/issues/1841
https://github.com/sourcegraph/sourcegraph/issues/1843

> Verify extension connects to the new instance & still works as expected

I verified it connected but didn't didn't have code intel setup on localhost so I didn't verify that.

I am going to close this tracking issue since it has been tested by two other issues. That said, this test plan seem generally applicable, so maybe it is worth committing as a checklist in the browser dir?
